docs/source/gateway/index.rst

```diff
@@ -356,6 +356,28 @@ Now that you have created several AI Gateway routes, you can create MLflow Model
 routes to build application-specific logic using techniques like prompt engineering. For more
 information, see :ref:`AI Gateway and MLflow Models <gateway_mlflow_models>`.
 
+Step 9: Set rate limits on AI Gateway routes
+--------------------------------------------------
+
+After you created AI Gateway routes, you can set rate limits on AI Gateway routes for cost control,
+ensuring production availability and fair sharing.  For example:
```

Comment:
Can we explain what kind of limit the following code sets and what happens when users reach that limit?

Reply:
Added!